packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -96,7 +97,20 @@ function CollabSidebarContent( { showCommentBoard, setShowCommentBoard } ) {
 			}
 		} );
 
-		return result;
+		const blockCommentIds = getBlocks()
```

Comment:
Will this work for comments in posts with a template, such as Site Editor > Pages?

Reply:
I have updated PR, it should be working now with post/page with template.